sheet-to-json.js

```diff
@@ -50,12 +53,37 @@ function sortObjByKey(value) {
     value;
 }
 
+function getDelta(state) {
+  return  rawData.raw_data.reduce((stat, row) => {
+    let stateName = row.detectedstate;
+    let isToday = moment().isSame(moment(row.dateannounced, "DD-MM-YYYY"), "day");
+    if (stateName && (stateName === state || state === "Total") && isToday) {
+      let currentStatus = row.currentstatus;
+      if (currentStatus) {
+        stat.confirmed += 1;
+        switch (currentStatus) {
```

Comment:
@sumitmckv I see the following status:
```
Recovered
Hospitalized
Deceased
Migrated
```
Include `Migrated` as well. This field should be included on https://api.covid19india.org/data.json `statewise` too

Reply:
what does Migrated signify? 
The sum of recovered, hospitalized and deceased should be total confirmed cases. Introduction of Migrated will have that maths messed up.
Also the main table on website shoes confirmed, active, recovered and deceased. Not sure which bucket the migrated data will fall into.

Reply:
I do not know about `Migrated` status. I see this status on the `raw_data.json`.
So thought of including it.

```json
{
  "agebracket": "",
  "backupnotes": "Travelled to Singapore (Feb) and Canada (Mar)",
  "contractedfromwhichpatientsuspected": "",
  "currentstatus": "Migrated",
  "dateannounced": "17/03/2020",
  "detectedcity": "Saket",
  "detecteddistrict": "",
  "detectedstate": "Delhi",
  "estimatedonsetdate": "",
  "gender": "",
  "nationality": "India",
  "notes": "Travelled from Singapore (Feb) and Canada (Mar)",
  "patientnumber": "130",
  "source1": "http://health.delhigovt.nic.in/wps/wcm/connect/a50f1c804d973055ae77ff982ee7a5c7/HB17.pdf?MOD=AJPERES&lmod=-333510648&CACHEID=a50f1c804d973055ae77ff982ee7a5c7",
  "source2": "https://www.hindustantimes.com/delhi-news/delhi-s-8th-covid-19-case-flew-to-singapore-without-waiting-for-test-result/story-c4YFWwviCvti3TYnA124SM.html",
  "source3": "",
  "statepatientnumber": "DL-P8",
  "statuschangedate": "20/03/2020"
}
```